README.md

```diff
@@ -84,6 +84,15 @@ commands that might fit your system:
 - Ubuntu    `apt install libclang-dev libspeechd-dev speech-dispatcher speech-dispatcher-espeak espeak`
 - Arch      `pacman -S speech-dispatcher espeak`
 
+### Compile without spellchecking
+
+Some users may encounter issues building the spellcheck feature on ARM64. To
```

Comment:
Be more specific here. It's not ARM64 it's Mac ARM64. This chip has caused me a lot of headaches so let's not "not point fingers". :smile: